app/components/elements/Dialog.js

```diff
@@ -24,13 +24,20 @@ export function DialogTitle(props) {
     ...dialogTitleProps
   } = props;
 
+  const ICON_WIDTH = 28;
+
   return (
     <Flex
       p={3}
-      sx={{ alignItems: 'center', justifyContent: 'space-between', borderBottom: props.divider ? borders.divider : 'unset', ...sx }}
+      sx={{ alignItems: 'center', borderBottom: props.divider ? borders.divider : 'unset', ...sx }}
       {...dialogTitleProps}
     >
-      {children}
+      <Flex 
+        mx={[0, 'auto']} 
+        pl={closeIcon ? ICON_WIDTH : 0} // ensure Title is properly centered when icon present
```

Comment:
This is somewhat a hack to offset the width of the icon when trying to center the title
![image](https://github.com/user-attachments/assets/a2b21954-8cd1-4d78-84c5-6ef87cc18032)

I couldn't think of a better way to easily do it. Typically I would set the icon position to absolute, but I was unable to make that happen with the `sx` prop, I don't think it is supported.